package/network/services/openvpn/files/openvpn.init

```diff
@@ -105,26 +105,31 @@ start_instance() {
 start_service() {
 	. /usr/share/openvpn/openvpn.options
 	config_load 'openvpn'
-	config_foreach start_instance 'openvpn'
 
-	local path name
-	for path in /etc/openvpn/*.conf; do
-		if [ -f "$path" ]; then
-			name="${path##*/}"; name="${name%.conf}"
+	if [ -n "$1" ]; then
+		start_instance "$1"
```

Comment:
A check is missing verifying if the specified instance name is actually a valid openvpn uci section.
Now start_instance can be called with an invalid instance name which is not desired.
Have a look at dnsmasq init script how this is handled (https://git.lede-project.org/?p=source.git;a=blob;f=package/network/services/dnsmasq/files/dnsmasq.init;h=3d3d83334e0073b1092320a25a553513ca25c3db;hb=HEAD#l1081)